playbook/config/initializers/global_variables.rb

```diff
@@ -0,0 +1,2 @@
+MENU = YAML.load_file("#{Playbook::Engine.root}/app/pb_kits/playbook/data/menu.yml")
+SAMPLES = YAML.load_file("#{Playbook::Engine.root}/app/pb_kits/playbook/data/samples.yml")
```

Comment:
@thestephenmarshall How do you feel about the naming conventions here?  For the global variables themselves (`MENU` & `SAMPLES`) and for the initializer file (`global_variables.rb`).